fabtests/multinode/src/harness.c

```diff
@@ -60,16 +60,16 @@ static int parse_caps(char *caps)
 static int parse_pattern(char *pattern)
```

Comment:
Change function to return enum multi_pattern.  And shouldn't there be a change where this function returns to to check for the enum values?